akka-http-core/src/main/resources/reference.conf

```
@@ -556,6 +556,21 @@ akka.http {
     # will automatically terminate itself. Set to `infinite` to completely disable idle timeouts.
     idle-timeout = 30 s
 
+    # The time after which an idle connection (without pending requests)
+    # will automatically terminate itself. Set to `infinite` to completely disable idle timeouts.
+    #
+    # Note that this is different than the idle-timeout which is about the whole pool.
+    #
+    # This settings afects each single connection independently.
+    # In that context, idle means the connection is sitting unused in the pool and is not associated any ongoing request.
+    #
+    # A common scenario where this setting can be useful is when a host closes a connection according to its keep-alive
+    # timeout and at the same time the client tries to sent a request on a now invalid connection.
+    # Such a call will fail with UnexpectedConnectionClosureException.
+    # This can be prevented (mitigated) by configuring this timeout with a value lower than the host keep-alive timeout.
+    # As such, you minimise the risk of using a connection that is likely to be closed by the host.
```

IIRC we use American style usually

```suggestion
    # As such, you minimize the risk of using a connection that is likely to be closed by the host.
```